src/extensions/windows/Microsoft.DotNet.UpgradeAssistant.Extensions.Windows/PackageMaps/UwpPackageMap.json

```diff
@@ -15,7 +15,7 @@
     "NetCorePackages": [
       {
         "Name": "Microsoft.WindowsAppSDK",
-        "Version": "1.0.0"
+        "Version": "1.1.0-preview1"
```

Comment:
Any chance 1.1.0 works? I'd like to avoid depending on preview versions as they rev so frequently and (generally) aren't guaranteed to have stable APIs.

Reply:
Windows App SDK 1.1.0 is not out yet.

Reply:
Is this change needed to fix the bug in the associated issue?

Reply:
1.1.0 is finally out. I have updated the version